src/build/verification.ts

```diff
@@ -1,14 +1,16 @@
 import { existsSync } from 'node:fs'
+import { readFile } from 'node:fs/promises'
 import { join } from 'node:path'
 
+import { glob } from 'fast-glob'
 import { satisfies } from 'semver'
 
 import { ApiRouteType, getAPIRoutesConfigs } from './advanced-api-routes.js'
 import type { PluginContext } from './plugin-context.js'
 
 const SUPPORTED_NEXT_VERSIONS = '>=13.5.0'
 
-const warnings = new Set<string>()
+const verifications = new Set<string>()
```

Comment:
NIT: should we put this on `PluginContext` object instead of module scope maybe? this forces the use of `resetModules` which does work, but feels wrong to use for code we own if we could avoid that?

Reply:
i attempted to do this in various ways but ended up down a rabbit hole, because it didn't seem right to store the verifications state in the plugin context (and mutate it directly from the verifications file) as it doesn't have anything to do with the rest of the stuff in the plugins context. I'd like to handle verifications differently (maybe creating a class that can store its own state?) but that's for a follow up

Reply:
That's ok, this is not a blocker to me